site/en/guides/border-router/_toc.yaml

```diff
@@ -1,21 +1,28 @@
 toc:
 - title: Overview
   path: /guides/border-router/
-- heading: Platforms - Linux
+- title: Get Started
+  path: /guides/border-router/get-started
+
+- heading: Platforms 
```

Comment:
Can we keep the Linux identifier here so it's clear

Reply:
Sure, done. I think I had a plan for this...then changed my mind & forgot to add it back.